packages/playwright-core/src/server/registry/index.ts

```diff
@@ -1248,9 +1251,29 @@ export class Registry {
     }
   }
 
-  private async _validateInstallationCache(linksDir: string) {
-    // 1. Collect used downloads and package descriptors.
-    const usedBrowserPaths: Set<string> = new Set();
+  async list() {
+    const linksDir = path.join(registryDirectory, '.links');
+    const browsers: Array<{ browserName: string, browserVersion: number, hostDir: string, browserPath: string }> = await this._traverseBrowserInstallations(linksDir);
```

Comment:
You shouldn't need explicit type declaration for `const browsers` here. Does `tsc` fail to derive the type here?

Reply:
No. tsc was able to derive the type. Removed uneccessary type declarations here.